### ISSUE TITLE
Unable to run rector since v0.10.20 (missing dependencies)

### ISSUE THREAD

**Issue by liarco** [rectorphp/rector#6406]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.10.20+              |
| Installed as   | composer dependency                           |

How to reproduce:
```bash
> mkdir tmp && cd tmp
> composer require rector/rector
...
> php vendor/bin/rector --help

[ERROR] Class 'Symplify\ComposerJsonManipulator\Bundle\ComposerJsonManipulatorBundle' not found

> composer require symplify/composer-json-manipulator
> php vendor/bin/rector --help
Description:
  Upgrade or refactor source code with provided rectors

Usage:
  process [options] [--] [<source>...]

...
```

What am I missing?

Thank you for your time.

**Comment by TomasVotruba** [comment]
Hi,
thanks for reporting.

This should help: https://github.com/rectorphp/rector-src/commit/d2b8907ce6fb86dd4d8fe2dd661f348b97e812e3

Released under 0.10.22: https://github.com/rectorphp/rector-src/tree/0.10.22

**Comment by liarco** [comment]
Thank you very much, I tested `v0.10.22` and it works as intended! 😃
